### PR TITLE
Assortativity, Betweeness centrality, Clustering and ClusterinAV

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Review by annacanal — COMMENTED** [review]
Merge the updates from gv-genesis-2 and adapt the tests. In general all looks good

**Comment by c-yuwei** [comment]
Is clustering compatible with multiplex?

**Comment by annacanal** [comment]
> Is clustering compatible with multiplex?

Yes it is. One person might want to know the clustering within each layer.
Like Strength is compatible with multiplex. It will give a result per each layer

File: braph2genesis/measures/_Transitivity.gen.m [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+MultiplexGraphBU
+%%%% ¡code!
+A11 = [
+      0 1 1 1;
+      1 0 1 0;
+      1 1 0 1;
+      1 0 1 0
+      ];
+A12 = eye(4);
+A21 = eye(4);
```
Comment (c-yuwei) [inline braph2genesis/measures/_Transitivity.gen.m:95]: Adapt for new multiplex format implementation.

File: braph2genesis/measures/_Clustering.gen.m [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+scope = Measure.UNILAYER;
+
+%%% ¡parametricity!
+parametricity = Measure.NONPARAMETRIC;
+
+%%% ¡compatible_graphs!
+GraphWU
+GraphWD
+GraphBU
+GraphBD
```
Comment (annacanal) [inline braph2genesis/measures/_Clustering.gen.m:23]: it is also compatible to Multiplex graphs.

            list = { ...
                'GraphBD' ...
                'GraphBU', ...
                'GraphWD', ...
                'GraphWU' ...
                'MultiplexGraphBD', ...
                'MultiplexGraphBU', ...
                'MultiplexGraphWD' ...
                'MultiplexGraphWU' ...
                };

File: braph2genesis/measures/_Assortativity.gen.m [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+%%% ¡scope!
+scope = Measure.UNILAYER;
+
+%%% ¡parametricity!
+parametricity = Measure.NONPARAMETRIC;
+
+%%% ¡compatible_graphs!
+GraphWU
+GraphBU
+
```
Comment (annacanal) [inline braph2genesis/measures/_Assortativity.gen.m:22]: also compatible to 
                'MultiplexGraphBU', ...
                'MultiplexGraphWU' ...

File: braph2genesis/measures/_BetweennessCentrality.gen.m [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+%%% ¡parametricity!
+parametricity = Measure.NONPARAMETRIC;
+
+%%% ¡compatible_graphs!
+GraphWU
+GraphWD
+GraphBU
+GraphBD
+
```
Comment (annacanal) [inline braph2genesis/measures/_BetweennessCentrality.gen.m:23]: also compatible to 
                'MultiplexGraphBD', ...
                'MultiplexGraphBU', ...
                'MultiplexGraphWD', ...
                'MultiplexGraphWU' ...

File: braph2genesis/measures/_ClusteringAv.gen.m [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+scope = Measure.UNILAYER;
+
+%%% ¡parametricity!
+parametricity = Measure.NONPARAMETRIC;
+
+%%% ¡compatible_graphs!
+GraphWU
+GraphWD
+GraphBU
+GraphBD
```
Comment (annacanal) [inline braph2genesis/measures/_ClusteringAv.gen.m:21]: same also compatible to all multiplex graphs

**Review by annacanal — CHANGES_REQUESTED** [review]
You are missing compatible multiplex graphs on the measures.

**Review by annacanal — APPROVED** [review]
I updated all the measures to work with multiplex graphs. Check them

**Review by c-yuwei — APPROVED** [review]

File: braph2genesis/measures/_BetweennessCentrality.gen.m [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+%%% ¡compatible_graphs!
+GraphWU
+GraphWD
+GraphBU
+GraphBD
+MultiplexGraphBD
+MultiplexGraphBU
+MultiplexGraphWD
+MultiplexGraphWU
+%% ¡props_update!
```
Comment (giovannivolpe) [inline braph2genesis/measures/_BetweennessCentrality.gen.m:27]: leave space

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
update all:
% adjacency matrix (for graph) or 2D-cell array (for multigraph, multiplex, etc.)

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**